docker-compose-native/docker-compose.yml

```diff
@@ -48,9 +57,9 @@ services:
       - ./prestissimo/coordinator/config.properties:/opt/presto-server/etc/config.properties
       - ./prestissimo/coordinator/jvm.config:/opt/presto-server/etc/jvm.config
       - ./prestissimo/coordinator/node.properties:/opt/presto-server/etc/node.properties
-      - ./prestissimo/catalog/jmx.properties:/opt/presto-server/etc/catalog/jmx.properties
-      - ./prestissimo/catalog/tpch.properties:/opt/presto-server/etc/catalog/tpch.properties
-      - ./prestissimo/catalog/hive.properties:/opt/presto-server/etc/catalog/hive.properties
+      - ./prestissimo/coordinator/catalog/hive.properties:/opt/presto-server/etc/catalog/hive.properties
+      - ./prestissimo/coordinator/catalog/tpcds.properties:/opt/presto-server/etc/catalog/tpcds.properties
```

Comment:
native has no tpcds support yet.

Reply:
Yes I had just added it because it would be supported soon but it is better to have just the hive connector, removed additional connectors.